src/actions.ml

```diff
@@ -2288,3 +2288,88 @@ let coq_check_stale_pr ~bot_info ~owner ~repo ~after ~throttle =
     >>= fun () -> Lwt.return true
   in
   apply_after_label ~bot_info ~owner ~repo ~after ~label ~action ~throttle ()
+
+let run_bench ~bot_info comment_info =
+  (* Do we want to use this more often? *)
+  let open Lwt.Syntax in
+  let owner = comment_info.issue.issue.owner in
+  let repo = comment_info.issue.issue.repo in
+  let pr_number = comment_info.issue.number in
+  (* We need the GitLab build_id and project_id. Currently there is no good way
+     to query this data so we have to jump through some somewhat useless hoops in
+     order to get our hands on this information. TODO: do this more directly.*)
+  let* gitlab_check_summary =
```

Comment:
Why `let *`?

Reply:
That comes from `Lwt.Syntax`, it just does `Lwt.bind` i.e. `>>=`. This let's you write monadic code slightly more cleanly, but here it hasn't really gained us much. The trouble is we only have `let*` for `Lwt.t`, but we would also need something similar for `Result.result`, but I didn't really want to have to use `ppx_let`.